vm/src/vm.rs

```diff
@@ -936,7 +936,7 @@ impl VirtualMachine {
         }
 
         let attr = if let Some(ref dict) = obj.dict {
-            dict.get_item_option(name_str.clone(), self)?
+            dict.get_item_option(&name_str.value, self)?
```

Comment:
```suggestion
            dict.get_item_option(name_str.as_object().clone(), self)?
```

Reply:
Note that this will create a full PyObject, and be less efficient than the `String` implementation of the hashing.

Reply:
Ah, never mind then.